### ISSUE TITLE
pkdeploy broken: PyPI changed interface

### ISSUE THREAD

**Issue by robnagler** [radiasoft/pykern#17]
[Travis 147 failed](https://travis-ci.org/radiasoft/pykern/builds/249804913) with:

```text
Submitting .tox/dist/pykern-20170703.223032.zip to https://pypi.python.org/pypi
Upload failed (410): Gone (This API has been deprecated and removed from legacy PyPI in favor of 
using the APIs available in the new PyPI.org implementation of PyPI (located at https://pypi.org/). 
For more information about migrating your use of this API to PyPI.org, please see 
https://packaging.python.org/guides/migrating-to-pypi-org/#uploading. For more information about
 the sunsetting of this API, please see 
https://mail.python.org/pipermail/distutils-sig/2017-June/030766.html)
```

**Comment by robnagler** [comment]
Still broken

**Comment by robnagler** [comment]
Fix 9cc10bab0a293a1ccba654e2d869f2b8aba59392
